README.md

```diff
@@ -41,6 +41,28 @@ SupplyAlly helps ensure that everyone can get their fair share of items. By scan
    - `npm run storybook`: If you'd like to use Storybook for development
 8. Open Expo and launch the app
 
+9. To fix the network issues on expo app in terms of backend api not reachable :
```

Comment:
Let's add this to a `Troubleshooting` section instead, so we can include a list of issues that we've encountered before

Reply:
added